### PR TITLE
docs(spinnaker): add Mercari to user list

### PR DESCRIPTION
**Pull request by tcnksm**
[Mercari](https://www.mercari.com/) is C2C marketplace app running in Japan, US, and UK. We use Spinnaker for CD of microservices on kubernetes (We talked about it on GCP event in Japan [slide](https://www.slideshare.net/GoogleCloudPlatformJP/microservices-at-mercari#34)). It would be great if we can be on the user list. Thank you for great tool!

### CODE REVIEW

**Comment by tcnksm** [comment]
>The following commits need their title changed:

Updated

File: community/index.md [hunk 1]
```diff
@@ -170,6 +173,7 @@ The Spinnaker OSS project was started at Netflix, and today thrives on the contr
     <div class="clearfix flex items-center mb3">
     {% include spinnaker_user id="weatherbug_user" %}
     {% include spinnaker_user id="xiaohongshu_user" %}
+    {% include spinnaker_user id="mercari_user" %}
```
Comment (tomaslin) [inline community/index.md]: the list is alphabetical so mercari should be between makerbot and mulesoft

Comment (tcnksm) [reply community/index.md]: Move it to between makerbot and mulesoft

**Review by dorbin — APPROVED** [review]
LGTM

**Comment by tomaslin** [comment]
thank you for your contribution

**Pull request merged**